docs/source/environments/index.rst

```diff
@@ -0,0 +1,360 @@
+FiftyOne Environments
+=====================
+.. default-role:: code
+
+This is a guide to using FiftyOne with data stored in various environments, for
+example, how to work with remote data.
+
+
+Terminology:
+
+* :ref:`Local Machine <local-data>` - Data is stored on the same machine on which FiftyOne
+  is installed
```

Comment:
I would maybe include something like this is a personal computer or laptop that has a Desktop/GUI to launch the App in.

FiftyOne will also need to be install on remote machines, which could cause confusion.